test/hack/cleanup/main.go

```diff
@@ -93,3 +105,26 @@ func main() {
 		}
 	}
 }
+
+func cleanUpCluster(ctx context.Context, name string, client *eks.Client, log *zap.SugaredLogger, expirationTime time.Time) {
+	if name != "" {
+		if strings.HasPrefix(name, "soak-") {
```

Comment:
needs to be this, right?
```suggestion
		if strings.HasPrefix(name, "soak-periodic") {
```